src/tests/pages/apps.test.tsx

```diff
@@ -84,145 +42,291 @@ describe('AppsPage', () => {
+      })
+
+      // unpin Transaction Builder Safe App
+      fireEvent.click(screen.getByLabelText('Unpin Transaction Builder'))
+
+      // show Bookmarked Safe Apps only
+      await waitFor(() => {
+        expect(screen.queryByText('Compound')).toBeInTheDocument()
+        expect(screen.queryByText('Transaction Builder')).not.toBeInTheDocument()
+      })
+    })
+
+    it('shows Safe app details when you click on the Safe app card', async () => {
+      // mock 2 Bookmarked Safe Apps
+      const mockedBookmarkedSafeApps = {
+        137: { pinned: [compopundSafeAppMock.id, transactionBuilderSafeAppMock.id] },
+      }
+
+      window.localStorage.setItem(`${LS_NAMESPACE}safeApps`, JSON.stringify(mockedBookmarkedSafeApps))
+
+      render(<BookmarkedSafeAppsPage />, {
+        routerProps: {
+          pathname: '/apps/bookmarked',
+          query: {
+            safe: 'matic:0x0000000000000000000000000000000000000000',
+          },
+        },
+      })
+
+      // drawer is not present
+      expect(screen.queryByRole('presentation')).not.toBeInTheDocument()
+
+      // clicks on Transaction Builder Safe App
+      await waitFor(() => {
+        fireEvent.click(screen.getByRole('heading', { level: 5, name: 'Transaction Builder' }))
+      })
+
+      await waitFor(() => {
+        const safeAppPreviewDrawer = screen.getByRole('presentation')
+        expect(safeAppPreviewDrawer).toBeInTheDocument()
+        // Transaction Builder Safe App title
+        expect(getByRole(safeAppPreviewDrawer, 'heading', { level: 4, name: 'Transaction Builder' }))
+        // open app button should be present
+        expect(getByText(safeAppPreviewDrawer, 'Open App'))
+      })
+    })
+  })
+
+  describe('Custom Safe apps Page', () => {
+    it('shows Custom safe apps section', async () => {
+      render(<CustomSafeAppsPage />, {
+        routerProps: {
+          pathname: '/apps/custom',
+          query: {
+            safe: 'matic:0x0000000000000000000000000000000000000000',
+          },
+        },
+      })
+
+      await waitFor(() => {
+        // show add custom app card
+        expect(screen.getByRole('button', { name: 'Add custom app' }))
+      })
+
+      // Add custom app modal is not present
+      expect(screen.queryByRole('presentation')).not.toBeInTheDocument()
+
+      await act(() => {
+        fireEvent.click(screen.getByRole('button', { name: 'Add custom app' }))
+      })
+
+      // shows Add custom app modal
+      await waitFor(() => {
+        expect(screen.getByRole('heading', { level: 2, name: 'Add custom app' })).toBeInTheDocument()
+
+        // shows custom safe App App Url input
+        const customSafeAppURLInput = screen.getByLabelText(/App URL/)
+        expect(customSafeAppURLInput).toBeInTheDocument()
       })
     })
 
-    it('allows adding custom apps', async () => {
+    it('Adds a Custom Safe App', async () => {
       const APP_URL = 'https://apps.gnosis-safe.io/compound'
```

Comment:
There are still some elements using the old domain. As this is a mock shouldn't cause issues but just in case maybe it's better to update. Also in terms of marketing

Reply:
done! :3